contracts/multisig-improved/src/action_types/propose.rs

```diff
@@ -73,11 +73,16 @@ pub trait ProposeModule:
         let own_shard = self.blockchain().get_shard_of_address(&own_sc_address);
         match action {
             Action::SendTransferExecuteEgld(call_data) => {
-                let other_sc_shard = self.blockchain().get_shard_of_address(&call_data.to);
                 require!(
                     call_data.egld_amount > 0 || !call_data.endpoint_name.is_empty(),
                     "proposed action has no effect"
                 );
+
+                if cfg!(debug_assertions) {
```

Comment:
What does this do? At a SC level?

Reply:
https://doc.rust-lang.org/rust-by-example/attribute/cfg.html

The `cfg!` macro allows for checks at runtime